src/components/site/EmphasizedImage.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const EmphasizedImage = props => (
+  <div className="dg_card-image" style={props.backgroundimage}>
```

Comment:
I would recommend moving `props.backgroundImageSrc` to a variable.

```js
const cardImageStyles = {
  backgroundImage: props.backgroundImageSrc
};
```

then this line looks like

```js
<div className="dg_card-image" style={cardImageStyles}>
```

This allows you to only worry about passing the url of the image to the component.

Reply:
We may need to disregard this all together, as it doesn't seem like this is accessible. Can you try this out?

https://css-tricks.com/almanac/properties/o/object-fit/

This would include using hte physical image with an alt tag, instead of the background image.

See https://stackoverflow.com/a/48913760/1143670 for more details.

Reply:
Pushed the original solution over but that will change. Looking into other suggestions now.

Reply:
Followed the example and added an img tag with alternative text which should make it readable. This uses both the background image and the img tag is hidden but still accessible by assistive technologies.